app/graphql/types/profile.rb

```diff
@@ -44,7 +53,47 @@ class Types::Profile < Types::BaseObject
     null: true,
     description: 'The message this user has submitted for the Hall of Fame'
 
+  field :location, String,
+    null: true,
+    description: 'Your general location to be displayed on your profile.'
+
+  field :gender, String,
+    null: true,
+    description: 'What you identify as.'
+
+  field :birthday, GraphQL::Types::ISO8601Date,
+    null: true,
+    description: 'When you were born, or something like that...'
+
+  field :time_zone, String,
+    null: true,
+    description: 'The time_zone for this user.'
+
+  field :country, String,
+    null: true,
+    description: 'The country you are in currently.'
+
+  field :followers, Types::Profile.connection_type,
+    null: false,
+    description: 'The people the user follows'
+
+  field :following, Types::Profile.connection_type,
+    null: false,
+    description: 'The people the user is following'
+
   def url
     "https://kitsu/users/#{object.slug || object.id}"
   end
+
+  def followers
+    AssociationLoader.for(object.class, :followers).load(object).then do |follows|
+      follows.map(&:follower)
+    end
+  end
+
+  def following
+    AssociationLoader.for(object.class, :following).load(object).then do |follows|
+      follows.map(&:followed)
```

Comment:
N+1, please use a Loader

Reply:
I was going to ask, should I create a RecordLoader, I wasn't sure how to do through the AssociationLoader.